public/static/docs/user-guide/privacy.md

```diff
@@ -42,6 +42,13 @@ section of this User Guide.
 
 ## Scopes
 
+userinfo.email scope (view your email address), userinfo.profile scope (see your
```

Comment:
let's use `userinfo.email` (inline code) blocks for scope names to make a bit more readable

Reply:
@shcheklein have updated markup and have added mention about `drive.appdata` scope as well.
Not sure why CicrcleCI won't start (how it might be related to my GitHub account).

Reply:
@MaxRis don't worry about this for now. Have asked their support to check what the issue could be.